README.md

```diff
@@ -23,16 +22,27 @@ $ export EXPLORER=zzz
 
 4. **OPTIONAL** TLS or Onion
 
-#### TLS
+### TLS
+
+#### Trade interface
 
-Uncomment in the compose file (`docker-compose.yml`) the TLS related stuff and export ENV with the asbolute path to the SSL Certificate and Key to be used.
+To enable TLS encryption on the Trade interface of the daemon, you have to generate your own TLS key and certificate files. Once this is done, export the ENV variables with the absolute path of the files: 
 
 ```sh
-$ export SSL_CERT_PATH=/path/to/fullchain.pem
-$ export SSL_KEY_PATH=/path/to/privatekey.pem
+$ export SSL_CERT_PATH=/path/to/certificate.crt
+$ export SSL_KEY_PATH=/path/to/privatekey.key
 ```
 
-#### Onion
+The last step is to uncomment in the compose file (`docker-compose.yml`) the lines related to TLS for Trade interface.
+
+#### Operator interface
+
+By default, the daemon service is configured to enable TLS encryption for the Operator interface. It's enough to uncomment [this line](docker-compose.yml#L18) to disable it.
```

Comment:
Instead of using the exact line which can change in future with edits, directly mention the env variable name to uncomment